cloudfront/lambda/response_headers/dist/index.js.license

```diff
@@ -0,0 +1,158 @@
+@vercel/ncc
```

Comment:
is this necessery to store `dist` directory with compiled copy of code we already have in the codebase?

Reply:
This is so that the user of the module doesn't have to have node or npm installed or do any preprocessing to use it, they just need terraform. 

This is a somewhat similar approach to what github recommends for github actions.
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github